### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by andyljones** [andyljones/boardlaw#5]
All the hyperparameters right now were arrived at through 'grad student descent', which is to say they're garbage. Top of the list to tune:

* Main: buffer length, batch size, env count
* Opt: learning rate, [warmup duration](#4)... beta1 & beta2? Do I need momentum at large batch sizes?
* MCTS: sim count, c_puct, Dirichlet noise intensity and concentration
* [League](#4): field size, stable size, prime/challenger split, prior intensity  
* Network: width, depth.  

This should all wait until I've got a general algorithm I'm happy with though.

**Comment by andyljones** [comment]
I did a lot of tuning ad-hoc, and the current settings have been ossified by the fact I've done a bunch of experiments with them. Might want to vary them and do re-reuns in a future, better-resourced version of this project. Might also need to do some light variations to check we're approximately at optimal - the fact we hit quickly perfect play on 9x9 suggests we are, but reviewers will likely disagree.
